### DISCUSSION TITLE
Changing Themes & Menu Icon?

### DISCUSSION THREAD

**Discussion by Viavi91** [soulhotel/FF-ULTIMA#45] [Q&A]
Have tried changing my Windows mode from Dark/Light, have Firefox (fresh install) set on System Auto, nothing is changing from light to dark. I do have `toolkit.legacyUserProfileCustomizations.stylesheets = true` so I'm not sure what I'm missing. Trying remote debug as well is not showing any changes when editing. **EDIT**: This portion has been opened up as [Issue #46](https://github.com/soulhotel/FF-ULTIMA/issues/46#issue-2291393936)

An additional question, but how would I go about reverting the extensions icon back to default? Have tried removing the line as well as moving the images, and all it does is just make the icon disappear (though it is still clickable).

**Comment by soulhotel** [comment]
You probably only removed one of the lines for the extension menu, The icons changes are located in `all-global-theme-fullmoon` (dark mode) and `all-global-theme-dusky` (light mode). Lines below for you to easily see.

![firefox_YbswILYU9z](https://github.com/soulhotel/FF-ULTIMA/assets/155501797/6b21bf63-a608-427e-bec7-76392b572f70)

https://github.com/soulhotel/FF-ULTIMA/blob/e90b4a95e47db3fe535eb7e3bb98a3de43521f61/theme/all-global-theme-fullmoon.css#L464-L472
https://github.com/soulhotel/FF-ULTIMA/blob/e90b4a95e47db3fe535eb7e3bb98a3de43521f61/theme/all-global-theme-dusky.css#L474-L482

**Comment by soulhotel** [comment]
As for the the switching from dark to light mode. You shouldve opened this as an issue, but give your Firefox edition, Firefox Version, and OS.

As far as testing it, dark to light works on all OS's, linux distros, and ff 60+, but we'll figure out why it isnt for you.

**Reply by Viavi91** [reply]
Thanks for this - I'll mark this answered and bring that as an issue for continuity. My bad.
